package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@newrelic/newrelic-capacitor-plugin",
-  "version": "1.0.0",
+  "version": "1.2.0",
```

Comment:
Would you mind reverting this back to 1.1.0 since we are not doing a release for this right now? We follow [semantic versioning](https://semver.org/), so depending on the next release it might not be `1.2.0`